app/controllers/concerns/api/current/sync_to_user.rb

```diff
@@ -29,25 +29,40 @@ def processed_until(records)
     def response_process_token
       { current_facility_id: current_facility.id,
```

Comment:
We can check if `sync_from_beginning?` is true here and return an empty token from there itself.

Reply:
Makes sense.